app/controllers/admin/facility_groups_controller.rb

```diff
@@ -51,7 +58,11 @@ def destroy
   private
 
   def set_organizations
-    @organizations = policy_scope([:manage, :facility, Organization])
+    @organizations = if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+      current_admin.accessible_organizations(:manage).union(Organization.where(id: @facility_group&.organization_id))
```

Comment:
`union` the current facility_group's organization so an admin with facility_group level access can use the edit page